typescript-typing-tutor/main.ts

```diff
@@ -0,0 +1,38 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const spans = document.querySelectorAll('#phrase span');
+  let currentIndex = 0;
+
+  function updateCurrentSpan(): void {
```

Comment:
You don't need the 'DOMContentLoaded' event listener for this. You should remove lines 1 and 38 so that your variables and functions are in the global scope.